source/ShopPanel.h

```diff
@@ -202,15 +200,20 @@ class ShopPanel : public Panel {
 	bool SetScrollToBottom();
 	void SideSelect(int count);
 	void SideSelect(Ship *ship);
+	void MainAutoScroll(const std::vector<Zone>::const_iterator &selected);
 	void MainLeft();
 	void MainRight();
 	void MainUp();
 	void MainDown();
+	void CategoryAdvance(const std::string &);
```

Comment:
```suggestion
	void CategoryAdvance(const std::string &category);
```
we generally write the name of the variable in both places

Reply:
Yeah, don't know how I missed that one.